docs/Documentation.md

```diff
@@ -306,6 +365,7 @@ A default configuration file would look like
   "SpaceAroundDelimiter":true ,
   "KeepNewlineAfter":false,
   "MaxIfThenElseShortWidth":40,
+  "AlignBrackets": false,
```

Comment:
Could it be a more descriptive name maybe? `BracketsOnSeparateLines` or something?